Vagrantfile

```diff
@@ -150,6 +150,64 @@ def packages_netbsd
   EOF
 end
 
+def packages_cygwin
+  return <<-EOF
+    mkdir -p /cygdrive/c/cygwin
+    powershell -Command '$client = new-object System.Net.WebClient; $client.DownloadFile("https://www.cygwin.com/setup-x86.exe","C:\\cygwin\\setup-x86_64.exe")'
+    echo '
+    REM --- Change to use different CygWin platform and final install path
+    set CYGSETUP=setup-x86_64.exe
```

Comment:
oh, so you download the 32bit setup and store it under the name that usually the 64bit setup has? don't do that, that is confusing.